src/App.tsx

```diff
@@ -331,7 +331,12 @@ export default function App() {
         );
       }
       case View.STATS: {
-        return <Statistics></Statistics>;
+        return (
+          <Statistics
+            database={db}
```

Comment:
Instead of passing `database` as a prop, why not just use the `useDatabase` hook in `Statistics` (similar to `CardEditor`)?